app/controllers/interest_controller.rb

```diff
@@ -26,25 +26,15 @@ class InterestController < ApplicationController
   def list_interests
     store_location
     @title = :list_interests_title.t
-    notifications = find_relevant_notifications
-    interests = find_relevant_interests
-    @targets = notifications + interests
+    @interests = find_relevant_interests
 
     @pages = paginate_numbers(:page, 50)
-    @pages.num_total = @targets.length
-    @targets = @targets[@pages.from..@pages.to]
+    @pages.num_total = @interests.length
+    @interests = @interests[@pages.from..@pages.to]
   end
```

Comment:
Switches from `@targets` to more straightforward `@interests` for the view - we no longer have to add notifications to the interests.